packages/mobile/src/store/offline-downloads/sagas/offlineQueueSagas/workers/downloadCollectionWorker.ts

```diff
@@ -157,20 +154,20 @@ function* downloadCollectionAsync(
 }
 
 function* downloadCollectionCoverArt(collection: CollectionMetadata) {
-  const { cover_art_cids, cover_art_sizes, cover_art, playlist_id } = collection
-  const cid = cover_art_sizes ?? cover_art
-  const storageNodeSelector = yield* call(getStorageNodeSelector)
-
-  const imageSources = createAllImageSources({
-    cid,
-    endpoints: cid ? storageNodeSelector.getNodes(cid) : [],
-    size: SquareSizes.SIZE_1000_BY_1000,
-    cidMap: cover_art_cids
-  })
+  const { artwork, playlist_id } = collection
+
+  const primaryImage = artwork[SquareSizes.SIZE_1000_BY_1000]
+  if (!primaryImage) return
```

Comment:
Early return seems new here. Any downstream effects from ti?

Reply:
yeah the impact would be _not_ successfully off-lining something if it has no artwork... which is probably the right product impact, but let me look more closely here